lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -1559,8 +1559,8 @@ def test_progress_queries_paced_courses(self, self_paced, query_count):
 
     @patch.dict(settings.FEATURES, {'ASSUME_ZERO_GRADE_IF_ABSENT_FOR_ALL_TESTS': False})
     @ddt.data(
-        (False, 63, 46),
-        (True, 55, 40)
+        (False, 63, 44),
```

Comment:
This reduction is because of the usage of TieredCache, which saves cached data to both the request cache and the Django cache. The first number is the `initial` parameter and the second number is the `subsequent` parameter. The enhanced caching reduced the `subsequent` queries by two. It's not really predictable per unit - depends on what is happening in the call and what call follows another.